src/content/glossary/index.md

```diff
@@ -813,6 +977,10 @@ Short for "test network," a network used to simulate the behavior of the main Et
   Testnets
 </DocLink>
 
+### token {#token}
+
+A fungible virtual good that can be traded. More formally, a token is a database mapping addresses to numbers with the property that the primary allowed operation is a transfer of N tokens from A to B, with the conditions that N is non-negative, N is not greater than A’s current balance, and a document authorizing the transfer is digitally signed by A.
```

Comment:
> More formally, a token is a database mapping addresses to numbers with the property that the primary allowed operation is a transfer of N tokens from A to B, with the conditions that N is non-negative, N is not greater than A’s current balance, and a document authorizing the transfer is digitally signed by A.

This is too in the weeds for the glossary

Reply:
ok!